core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -657,6 +657,10 @@ class SparkContext(config: SparkConf) extends Logging with ExecutorAllocationCli
    *
    * Load data from a flat binary file, assuming the length of each record is constant.
    *
+   * '''Note:''' Normally getBytes returns an array padded with extra values,
```

Comment:
Hm, it's explicitly part of `FixedLengthBinaryInputFormat` in so far as that method constructs a bytes writable backed by a byte array of the given fixed record length ([here](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/input/FixedLengthBinaryRecordReader.scala#L111-123)). Does that count? In either case, I can expand the comment to avoid confusion.